src/components/Form/Form.jsx

```diff
@@ -39,79 +47,121 @@ const Form = () => {
 			});
 	}
 
+	useEffect(() => {
+		requiredData();
+		console.log(newEvent);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [newEvent]);
+
+	function requiredData() {
+		if (newEvent.title === '') {
```

Comment:
Perhaps this is also a case for a switch case

Reply:
Adjusted the error message, but didn't know how to make it a switch case